packages/tables/src/examples/overflow-menu.md

```diff
@@ -23,16 +23,6 @@ const OverflowMenu = ({ isHeader = false }) => (
       },
       flip: {
         enabled: false
-      },
-      offset: {
```

Comment:
@austin94 the offset needs to be 1 or 2px higher (for both headers and regular rows) in order to match the intended layout shown by https://garden.zendesk.com/css-components/tables/.

Reply:
I'm gonna get _really_ finicky here and show you what we want (I think you need to reapply a consistent negative offset to all menus in order to achieve the intended positioning). Please pay special attention to how high the top of the menu border is above the bottom border of the row (aka one pixel below the focused overflow trigger button).

### INCORRECT HEADER ROW

<img width="211" alt="screen shot 2018-10-12 at 12 00 30 pm" src="https://user-images.githubusercontent.com/143773/46889118-7ce8c780-ce17-11e8-9e57-8340f1f376d4.png">

### CORRECT HEADER ROW

<img width="218" alt="screen shot 2018-10-12 at 12 11 11 pm" src="https://user-images.githubusercontent.com/143773/46889295-05fffe80-ce18-11e8-9fd9-b4f89663f956.png">

### INCORRECT BODY ROW

<img width="209" alt="screen shot 2018-10-12 at 12 02 35 pm" src="https://user-images.githubusercontent.com/143773/46889156-9984ff80-ce17-11e8-8667-2bab8589f0c7.png">

### CORRECT BODY ROW

<img width="208" alt="screen shot 2018-10-12 at 12 11 26 pm" src="https://user-images.githubusercontent.com/143773/46889309-0ef0d000-ce18-11e8-989f-936cd7c23516.png">